### PR TITLE
*: merge hot stats

### PR DESCRIPTION
**Pull request by rleungx**
### What problem does this PR solve?

Issue Number: Ref #5839.

### What is changed and how does it work?

```commit-message
```

### Check List

Tests

- Unit test

### Release note

```release-note
None.
```

### CODE REVIEW

File: pkg/statistics/utils/constant.go [hunk 1]
```diff
@@ -12,17 +12,21 @@
...
 
 const (
-	// StoreHeartBeatReportInterval is the heartbeat report interval of a store.
-	StoreHeartBeatReportInterval = 10
-	// RegionHeartBeatReportInterval is the heartbeat report interval of a region.
-	RegionHeartBeatReportInterval = 60
+	// WriteReportInterval indicates the interval between write interval, the value is the heartbeat report interval of a region.
+	WriteReportInterval = 60
+	// ReadReportInterval indicates the interval between read stats report, the value is the heartbeat report interval of a store.
+	ReadReportInterval = 10
```
Comment (lhy1024) [inline pkg/statistics/utils/constant.go:25]: How about still using StoreHeartBeatReportInterval and RegionHeartBeatReportInterval?

File: pkg/statistics/utils/constant.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 TiKV Project Authors.
+// Copyright 2023 TiKV Project Authors.
```
Comment (lhy1024) [inline pkg/statistics/utils/constant.go:1]: Do we need to change it?

**Review by lhy1024 — COMMENTED** [review]
the rest LGTM

File: pkg/statistics/utils/constant.go (hunk 2 above)
Comment (rleungx) [reply pkg/statistics/utils/constant.go:1]: It's a new file.

File: pkg/statistics/utils/constant.go (hunk 1 above)
Comment (rleungx) [reply pkg/statistics/utils/constant.go:25]: Do you mean removing WriteReportInterval?

Comment (lhy1024) [reply pkg/statistics/utils/constant.go:25]: yes

**Review by bufferflies — APPROVED** [review]

**Review by lhy1024 — APPROVED** [review]

**Pull request merged**